cupy/core/core.pyx

```diff
@@ -2137,9 +2142,9 @@ cpdef ndarray moveaxis(ndarray a, source, destination):
         if not _has_element(src, n):
             order.push_back(n)
 
-    for i in range(len(src)):
-        n = <Py_ssize_t>i
-        order.insert(order.begin() + dest[n], src[n])
+    cdef Py_ssize_t d, s
+    for d, s in sorted(six.moves.zip(dest, src)):
```

Comment:
Sorry, my colleague says we need to use `six` in cython. Please use `zip`.

Reply:
No problem! I fixed it.